### PR TITLE
feat: improve vertical space usage

### PR DESCRIPTION
**Pull request by kof**
## Description

- all inputs/buttons have now 24px height
- horizontal paddings on all panels are 12px, vertical 8px - optimized for better vertical density
- drive-by: removed all old colors, now its all entirely based on typography collection from figma and so we can now make dark mode reality

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Comment by TrySound** [comment]
Maybe merge only colors and cleanup here? And then go with improving spaces.

**Comment by kof** [comment]
> Maybe merge only colors and cleanup here? And then go with improving spaces.

I was too dirty yesterday, it ended up being a single big mess, but its easier to merge all together than spend hours separating for the sake of clean prs

**Comment by TrySound** [comment]
I mean let's merge current stage before refactoring further

**Comment by kof** [comment]
will have it ready today

**Comment by TrySound** [comment]
More! More!

File: apps/builder/app/builder/features/style-panel/sections/outline/outline.tsx [hunk 1]
```diff
@@ -38,18 +38,16 @@ export const Section = () => {
           description={propertyDescriptions.outlineStyle}
           properties={["outlineStyle"]}
         />
-        <Box css={{ justifySelf: "end" }}>
```
Comment (TrySound) [inline apps/builder/app/builder/features/style-panel/sections/outline/outline.tsx:41]: This was done this way to make symmetric with borders section.

**Comment by TrySound** [comment]
Oh, again refactors and massive changes...

**Comment by TrySound** [comment]
Honestly this looks bloated like my Firefox 3.0 toolbars 15 years ago

image

**Comment by johnsicili** [comment]
1. Can fields like those in border be wider? Seems like a lot of wasted real estate.
Screenshot 2024-10-20 at 3 50 30 PM

2. Border style doesn't have a focus anymore when arrowing through (maybe from previous pr)

3. Default browser/webstudio styles are very light, for example in heading margins. I would make a shade darker.

All in all I don't have strong opinions on it. Not sure if it improves much.. maybe i can more without scrolling the style panel. It does feel cramped, but zero clue if that's because I'm just used to the current way.

**Comment by kof** [comment]
@johnsicili thank you, great feedback, agree with all of this

**Review by TrySound — APPROVED** [review]

**Pull request merged**